bundles/org.openhab.binding.doorbird/README.md

```diff
@@ -0,0 +1,239 @@
+# Doorbird Binding
+
+Binding for Doorbird D101 and D210x video doorbells.
+
+## Supported Things
+
+Two thing types are supported:
+
+- Doorbird D101
+- Doorbird D210x
+
+## Discovery
+
+Auto-discovery is not supported at this time.
+
+## Thing Configuration
+
+The following configuration parameters are available on the Doorbird thing:
+
+| Parameter | Required/Optional | Description |
+|-----------|-------------------|-------------|
+| ID | Required | The ID number of the Doorbird device. This is usually a single digit (e.g. 1). |
```

Comment:
You can easily format the tables with online tools like: http://markdowntable.com/

Reply:
See the discussion with @J-N-K [here](https://github.com/openhab/openhab2-addons/pull/5857#discussion_r316365507). I was trying avoid messing with alignment, which can be quite tedious. However, the link you provided makes it easy, so I'll replace what's here with the column-aligned version of the table.

Reply:
Done